numpy/core/getlimits.py

```diff
@@ -350,6 +350,7 @@ class finfo(object):
     --------
     MachAr : The implementation of the tests that produce this information.
     iinfo : The equivalent for integer data types.
+    sys.float_info : Python's definitions of these values for
```

Comment:
the "for" is dangling at the end, did you mean "Python's definition of these values." ?